server/meta.jsx

```diff
@@ -15,6 +15,9 @@ type SDKMeta = {|
     getSDKLoader : (options? : {| baseURL? : string, nonce? : string |}) => string
 |};
 
+const emailRegex = /^.+@.+$/;
```

Comment:
are we just looking for "roughly like an email" or do we want to use an RFC compliant regex?

Reply:
Talked a bit offline. Marking resolved.